src/components/CalendarMonth.jsx

```diff
@@ -25,6 +26,7 @@ const propTypes = forbidExtraProps({
   enableOutsideDays: PropTypes.bool,
   modifiers: PropTypes.object,
   orientation: ScrollableOrientationShape,
+  daySize: nonNegativeInteger,
```

Comment:
thats an omission in airbnb-prop-types, which i'll fix in the next semver-major :-)